src/lib/Translation/Extractor/JavaScriptFileVisitor.php

```diff
@@ -85,12 +87,14 @@ public function visitFile(SplFileInfo $file, MessageCatalogue $catalogue)
         }
 
         $ast->traverse(function ($node) use ($catalogue, $file) {
-            if ($this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_METHOD)) {
+            if ($this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_TRANS_METHOD) || $this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_TRANS_CHOICE_METHOD)) {
```

Comment:
CS: this becomes unreadable, even at github, without an ultrawide monitor ;)

Such long ifs can be multiline, if formatted in the following way: 
```suggestion
            if ($this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_TRANS_METHOD)
                || $this->isMethodCall($node, self::TRANSLATOR_OBJECT, self::TRANSLATOR_TRANS_CHOICE_METHOD)
            ) {
```